docs/lib/auth/fragments/native_common/signout/common.md

```diff
@@ -7,3 +7,9 @@ Calling signOut without any options will just delete the local cache and keychai
 
 <inline-fragment platform="android" src="~/lib/auth/fragments/android/signout/20_global_signout.md"></inline-fragment>
 <inline-fragment platform="ios" src="~/lib/auth/fragments/ios/signout/20_global_signout.md"></inline-fragment>
+
+Calling signout with `globalSignOut = true` will invalidate all the Cognito User Pool tokens of the signed in user. If the user is signed into a device, they won't be authorized to perform a task that requires a valid token when a global signout is called from some other device.
+
+<amplify-callout>
+Global signout functionality does not work if you use one of the web UI sign in methods.
```

Comment:
Can these customers do something to globally sign out a user?

Reply:
Not yet - that's the bug we talked about during GA launch we want to address as a fast follow.